docs/suse/deploy.md

```diff
@@ -0,0 +1,150 @@
+# Deploying Stratos
+
+The following instructions cover how to:
+
+- Deploy Stratos
+- Connect Kubernetes to Stratos to enable Kubernetes views in Stratos
+- Deploy Stratos Metrics
+- Connect Stratos Metrics to Stratos to enable Metrics views in Stratos
+
+These instructions assume that you are deploying the SUSE Cloud Application Platform and that you have deployed UAA/SCF using a `scf-config-values` configuration file.
+
+## Install Stratos with Helm
+
+To deploy Stratos, use the same `scf-config-values.yaml` file that you used to deploy SCF/UAA.
+
+Ensure you have a Storage Class configured and that is set to be the default.
+
+Install with:
+
+```
+helm install suse/console \
+    --name susecf-console \
+    --namespace stratos \
+    --values scf-config-values.yaml
+```
+
+Wait until all pods are in the Ready state.
+
+You can locate the IP and Port that Stratos is running on with:
+
+```
+kubectl get service stratos-ui-ext --namespace=stratos
+```
+
+This will give output similar to:
+
+```
+NAME             TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)                       AGE
+stratos-ui-ext   NodePort   172.24.239.140   10.17.3.1     80:30862/TCP,8443:32129/TCP   1h
+```
+
+> NOTE: When you log in to Stratos, your SUSE Cloud Foundry dpeloyment should already be registered and connected to Stratos.
+
+## Connecting Kubernetes to Stratos
+
+Stratos can show information from your Kubernetes environment.
+
+To enable this, you must register and connect your Kubernetes environment with Stratos.
+
+In the Stratos UI, go to `Endpoints` in the left-hand side navigation and click on the `+` icon in the top-right on the view - you should be shown the `Register new Endpoint` view.
+
+1. Select `Kubernetes` from the `Endpoint Type` dropdown
+1. Enter a memorable name for yor environment in the `Name` field
+1. Enter the URL of the API server for your Kubernetes environment
+1. Check the `Skip SSL validation for the endpoint` if using self-signed certificates
+1. Click `Finish`
+
+The view will refresh to show the new endpoint in the disconnected state.
+
+Next you will need to connect to this endpoint.
+
+1. In the table of endpoints, click the three-dot menu icon alongside the endpoint that you added above
+1. Click on 'Connect' in the dropdown menu
+1. Select the appropriate authentication type and provide the necessary values
```

Comment:
Did @irfanhabib write some docs on these?

Reply:
Not sure - I was only covering CaaSP for now - I could add notes for others